buildsystem/tools/lib/ExaGO.pm

```diff
@@ -12,10 +12,7 @@ use warnings;
 use v5.16;
 
 use ExaGO::Env;
-use ExaGO::ClangFormat;
-use ExaGO::CMakeFormat;
 use ExaGO::FileNamingConventions;
```

Comment:
This one too?

Reply:
Ah nvm I see that the yaml still calls this perl script?

Reply:
Yeah so the clang-format, cmake-format and autopep8 pipelines were super trivial, but the naming conventions one I had a lot more difficulty in fixing. I tried a whole bunch of things with regex but I am not wise enough in the ways of Python globbing and the configuration to make it worth the effort. For now I decided to keep naming conventions as-is